nullaway/src/main/java/com/uber/nullaway/qual/EnsuresNonNull.java

```diff
@@ -0,0 +1,19 @@
+package com.uber.nullaway.qual;
```

Comment:
Is this really a "Type **qual**ifier" in the Checker Framework sense?

Shouldn't this package be "com.uber.nullaway.annotation"? (cc: @msridhar )

p.s. But I am fine landing as is if not obvious what it should be named, please let's not spend months debating whether to call it annotation or annotation**s** 😄

Reply:
Switched to annotations.  4935c00